### PR TITLE
The one that updates .scss files to be ready for Dart Sass

### LINKED ISSUE #596: preparatory work to move to Dart Sass
After the [open space](https://github.com/springernature/frontend-open-space/issues/296) [discussion](https://springernature.slack.com/archives/C1HGRAQ8G/p1634821874000400) I've worked on seeing what, if anything, really breaks if we go with Dart Sass as the agreed Sass compiler for Springer Natures front-end toolkits.

Here's the list of changed files –
```bash
context/brand-context/default/scss/10-settings/colors/default.scss
context/brand-context/default/scss/10-settings/typography.scss
context/brand-context/default/scss/20-functions/colors.scss
context/brand-context/default/scss/abstracts.scss
context/brand-context/springer/scss/30-mixins/triangle.scss
toolkits/global/packages/global-card/scss/10-settings/default.scss
toolkits/global/packages/global-forms/scss/10-settings/_default.scss
toolkits/global/packages/global-forms/scss/50-components/_forms.scss
toolkits/global/packages/global-meta/scss/10-settings/_default.scss
toolkits/global/packages/global-meta/scss/10-settings/_nature.scss
toolkits/global/packages/global-meta/scss/10-settings/_springer.scss
toolkits/global/packages/global-meta/scss/50-components/_meta.scss
```
The main issue relates to how Dart Sass makes the `/` a CSS calc thing requiring any Sass use to include the Sass math module:

```scss
@use `sass:math`;
```
And rewrite the usage from:
```scss
$rounded: round($lightness / $context--greyscale-stops) * $context--greyscale-stops;
```
to include the calling of the relevant part of the Sass math module (`math.div()`):
```scss
$rounded: round(math.div($lightness, $context--greyscale-stops)) * $context--greyscale-stops;
```

Realistically, we can 'just switch' to Dart Sass now and things won't break. Your command line might just be full of warnings about things that need looking at.

I'll create a PR for the relevant changes to use `math:div()` in the `context` folders and will create issues for teams to do any `math:div()` changes too.

note: 
- some of these are because components require more files than I'd expect to compile correctly.
- there are some additions to file to compile correctly (see above point)
- there are also some inconsistencies in file naming (I'll address those in a separate issues).
- [ ] there could still be traps with mixins / functions that aren't used but available requiring attention.

### PR DESCRIPTION
**Pull request by sturobson**
After discussing moving from Node Sass, Libsass, or Ruby Sass to the canonical Dart Sass in [the open space ticket](https://github.com/springernature/frontend-open-space/issues/296) I have gone through the front-end toolkits repo to make any necessary changes.

Fortunately, for now, Dart Sass doesn't have any breaking changes but they will come. 

So, to pre-empt that and to move towards using the modules system that comes with Dart Sass I have made relevant changes where Dart Sass treats the `/` as something that CSS' calc function owns. 

Again, luckily, these were few and far between. This PR addresses those small issues.

To check this and test this I created a separate .scss config file for each brand and run them locally and fixed up any code to stop the warnings.

---

sidenote: I kept hitting an error when using the global-meta component and I don't think it's related to Dart Sass

```scss
Error: "8px is not defined in $context--spacing. Permitted values are 0, 2, 4, 8, 16, 24, 32, 48, 64."
  ╷
6 │         padding-#{$direction}: spacing($spacing);
  │                                ^^^^^^^^^^^^^^^^^
  ╵
  context/brand-context/default/scss/30-mixins/style.scss 6:26             u-keyline()
  toolkits/global/packages/global-meta/scss/50-components/_meta.scss 22:3  @import
  sass-springernature.scss 45:9                                            root stylesheet
```

It looks like the component has a Sass variable of `$meta--keyline-spacing` but the mixin being called is expecing the Sass variable to be called `$context--keyline-spacing`. 
If this is a different error I'm happy to open an issue then work on it to tidy it up. 

This closes #596

### CODE REVIEW

**Review by benjclark — DISMISSED** [review]

**Comment by alexkilgour** [comment]
Sorry not sure how I missed this. Looking now.

File: context/brand-context/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # History
 
+## 18.0.0 (2021-11-01)
```
Comment (alexkilgour) [inline context/brand-context/HISTORY.md:3]: You have added the new version here but haven't updated the `package.json` so it won't publish

File: toolkits/global/packages/global-card/package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-  "version": "3.3.0",
+  "version": "4.0.0",
   "license": "MIT",
   "description": "card component",
   "keywords": [
     "card"
   ],
   "author": "Springer Nature",
-  "brandContext": "^11.1.1"
+  "brandContext": "^16.0.0"
```
Comment (alexkilgour) [inline toolkits/global/packages/global-card/package.json:10]: Should this be `^18.0.0` for the dart sass version?

File: toolkits/global/packages/global-forms/package.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+	"version": "4.0.0",
 	"license": "MIT",
 	"description": "form component",
 	"keywords": [
 	  "css",
 	  "component"
 	],
   "author": "Jack Watkins",
-  "brandContext": "^6.0.0"
+  "brandContext": "^16.0.0"
```
Comment (alexkilgour) [inline toolkits/global/packages/global-forms/package.json:11]: Should this be ^18.0.0 for the dart sass version?

**Comment by sturobson** [comment]
Travis doesn't like the Sass and I'm thinking that's because of the Sass compiler in the front-end toolkit utilities still relying on Node Sass.

I've raised the start of a [PR for it here](https://github.com/springernature/frontend-toolkit-utilities/pull/68).

But I need help turning the Node-Sass syntax and implementation to use Dart Sass.

**Review by alexkilgour — DISMISSED** [review]
Great stuff. Let's merge this Monday and then start getting all the components up to date.

**Review by alexkilgour — APPROVED** [review]
Ready to go 🚀

**Pull request merged**